src/coffee/run.coffee

```diff
@@ -213,6 +212,7 @@ module.exports = class
         .then (authConfig) =>
           options =
             timeout: program.timeout
+            exportFormat: 'txt'
```

Comment:
Add output file format.

Reply:
should this really be a `txt` file?

Reply:
I chose a `txt` file extension because it's a log file.

Reply:
shouldn't it be a `log` file, then?